org.audacious_media_player.Audacious.yaml

```diff
@@ -0,0 +1,160 @@
+# WIP
+id: org.audacious_media_player.Audacious
+runtime: org.kde.Platform
+runtime-version: '5.14'
+sdk: org.kde.Sdk
+command: audacious
+finish-args:
+  - --device=dri
+  - --filesystem=xdg-music
+  - --own-name=org.atheme.audacious
+  - --share=ipc
+  - --share=network
+  - --socket=pulseaudio
+  - --socket=wayland
+  - --socket=x11
```

Comment:
```suggestion
  - --socket=fallback-x11
```

Reply:
Actually maybe not the best idea, forgot this is a Qt app now which has wayland issues sometimes.